app/Makefile

```diff
@@ -92,7 +92,6 @@ LINKFLAGS_eagle.app.v6 = 			\
 	-T$(LD_FILE) 				\
 	-Wl,@../ld/defsym.rom			\
 	-Wl,--no-check-sections 		\
-	-Wl,--wrap=_xtos_set_exception_handler	\
```

Comment:
Why do you remove the exception handler for unaligned loads?
I guess the firmware heavily relies on this and will fail miserably.

Reply:
> I can confirm that it is safe because Espressif have included unaligned access patch in their firmware core.

We did tweak our exception handler so that it plays nicely with the gdb stub, and this is essential IMO.  The _big_ problem with the nonOS stuff is that it is all closed source so you have to rely on disassembling library code to see what it really does.

Reply:
@TerryE Another option, rather than just stripping the handler out of our code like this PR does, is to filter it out of the libmain.a that's shipped by the SDK.  Seem viable?

Reply:
I will need to disassemble the `libmain.a` code just to make sure that it doesn't have other side-effects that we would need to preserve